docker-compose-dev.yml

```diff
@@ -0,0 +1,56 @@
+# dev compose
+# extend from docker-compose-common.yml
+geocoder:
+  extends:
+    file: docker-compose-common.yml
+    service: geocoder
+  links:
```

Comment:
I wish! :smiley: 

https://docs.docker.com/compose/extends/#reference

> Compose copies configurations from the original service over to the local one, except for links and volumes_from. These exceptions exist to avoid implicit dependencies—you always define links and volumes_from locally. This ensures dependencies between services are clearly visible when reading the current file. Defining these locally also ensures changes to the referenced file don’t result in breakage.